db/pre-re/skill_cast_db.txt

```diff
@@ -1410,7 +1410,7 @@
 2325,2000,2000,0,30000:45000:60000:75000:90000,0,540000:480000:420000:360000:300000
 
 //-- LG_KINGS_GRACE
-5013,1000,0,0,5000,0,0
+5013,1000,0,0,5000,0,60000,-1
```

Comment:
This is wrong, there shouldn't be a 8th value as that only applies to renewal for fixed cast.